### DISCUSSION TITLE
Tagesertrag Nullen mit OpenDTU on Battery

### DISCUSSION THREAD

**Discussion by groovepack** [hoylabs/OpenDTU-OnBattery#1277] [Q&A] [answered]
Hallo,

erst mal vielen Dank an alle, die dieses Projekt ermöglicht haben und aktiv betreuen.

Meine Anlage: 4 Panele (1.8kWp) mit Victron Laderegler, Pylontech 3000 und Hoymiles HMS-800-2T.

Das Projekt habe ich eben auf ein Fusion Board (16MB) aktualisiert und dort die Firmware 2024.09.11 aufgespielt, vorher war es ein Board von Lukas und Funkmodul mit FW 2024.06.02

Leider funktioniert seit der Umstellung das Nullen des Tagesertrages nicht mehr. Egal was ich im Wechselrichter einstelle, der Tagesertrag wird nicht um Mitternacht zurückgestellt.

Hier die aktuelle Einstellung im WR, genau wie im alten Setup:

<img width="790" alt="image" src="https://github.com/user-attachments/assets/40025b74-a57b-4bc7-b33b-356a5e9f60e7">

Im Info-Layer steht, dass dies nur funktioniert, wenn der Wechselrichter nicht ereichbar ist, was er aber durch den Akku ist und nur =< 10% auf 0 gestellt wird. Übersehe ich etwas, was ich noch aktivieren muss?

Vielen Dank für Euren TIpp!

Matthias

**Comment by spcqike** [comment]
War das Thema nicht letztens bereits irgendwo? Und hängt die Lösung nicht am Neustart des wechselrichters? 

Die nullung dort deaktivieren und den wechselrichter um Mitternacht Neustarten lassen. Irgendwie so glaube ich (:

**Comment by groovepack** [comment, marked as answer]
Danke @spcqike - Du hast mich auf die richtige Spur gebracht. Ich habe da tatsächlich mehrmals drübergelesen.

Unter Einstellungen > Dynamic Power Limiter > Section "Wechselrichter" gibt es die Option "Uhrzeit für geplanten Neustart.

Gestern eingerichtet und siehe da, heute waren die Werte zurückgesetzt.

**Reply by JoSoBay** [reply]
Hallo, ich habe das gleiche Problem mit der Nullung des Tagesertrags. Nur finde ich bei mir nicht die Option für die Uhrzeit für geplanten Neustart. Kannst du vielleicht einen Screenshot von der Option zeigen. Habe die 2024.9.11 Firmwareversion.

<img width="1194" alt="image" src="https://github.com/user-attachments/assets/3e6fe81e-8f96-4a7f-a27f-c44a9ca7f7ea">

Vielen Dank

**Reply by spcqike** [reply]
Wechselrichter, die von Solar Paneelen gespeist werden, brauchen keinen erzwungenen Neustart. Da diese nachts ohne Sonnenschein aus sind. Daher ist die Option in dem Fall nicht sichtbar.

**Reply by JoSoBay** [reply]
Ja so habe ich es auch seit über einem Jahr betrieben. Mit reinem Solarbetrieb geht die Nullung. Jetzt betreibe ich eine Anker Solarbank an einem HMS-800 und damit geht die Nullung nicht mehr, da der Wechselrichter ja Strom vom Akku bekommt. Dann mache ich es halt über einen Neustart der Stromsteckdose. Dachte nur das der geplante Neustart immer angezeigt wird und ich das nur Übersehen habe.

**Reply by groovepack** [reply]
Unter Einstellungen > Dynamic Power Limiter findest Du den Punkt

![image](https://github.com/user-attachments/assets/7e192125-5184-4bf2-b70c-654260436ce9)

**Comment by Cla30** [comment]
@groovepack 
Wie hast du den die Anschlüsse hinbekommen auf dem neuen Fusions Board? Habe auch die alte onbattery und würde gerne auf Fusion umsteigen aber mir ist noch nicht klar wie ich die Pylontech und die Victron ans Board bekomme. 
Lieben Dank

**Reply by Sorbat** [reply]
Kennst du das? https://shop.allianceapps.io/products/opendtu-fusion-can-iso-shield

**Reply by Cla30** [reply]
Ja die Erweiterung kenne ich. Aber bei der alten habe ich zwei Stecker einen Netzwerk Stecker für die Pylontech und einen für die Victron. Hier habe ich nur klemmen.

**Reply by groovepack** [reply]
Sorry, irgendwie die Benachrichtigung dazu übersehen und hoffe die Antwort hilft Dir noch.

Ja, fertige Stecker wären eine feine Sache, aber da musst Du leider die Kabel "opfern".

Beim Cat-Netzwerk-Kabel sind Blau und Blau/Weiß und für den Victron musst Du mit Deinen Farben vergleiche, weil wohl nicht standardisiert.

![Fusion-Adapert](https://github.com/user-attachments/assets/30a8c555-d4f9-4bb3-a788-5894edf670a5)
![Victron](https://github.com/user-attachments/assets/e385270f-5037-4715-b47e-605080df6cae)
